documentation/examples/interconnection_study_example.json

```diff
@@ -1,10 +1,10 @@
 {
     "file_format_version": 1.1,
-    "site": "lga02",
-    "client_provider": "comcast",
-    "start_time": "2014-02-01T00:00:00Z",
+    "site": ["lga01", "lga02"],
```

Comment:
In the selector, but internally (main.py and onward) I think we should keep them singular.